doc/releases/changelog-dev.md

```diff
@@ -413,6 +413,12 @@
 
 <h3>Breaking changes 💔</h3>
 
+* Decompositions of two-qubit unitaries via `two_qubit_decomposition` now use 3 CNOTs even if
```

Comment:
Is `two_qubit_decomposition` a user-facing function? 🤔

Reply:
It is public and visible in the documentation, not sure if anyone actually uses it though.

Reply:
The [compilation hub entry on two-qubit synthesis](https://pennylane.ai/compilation/two-qubit-synthesis) advertises it, for example.